session/pessimistic_test.go

```diff
@@ -748,10 +748,13 @@ func (s *testPessimisticSuite) TestInnodbLockWaitTimeoutWaitStart(c *C) {
 		}()
 		_, err = tk2.Exec("select * from tk where c1 = 1 for update")
 	}()
+	time.Sleep(time.Millisecond * 30)
```

Comment:
Is this necessary?

Reply:
There may be sometime before the tk2 statement reach `failpoint` code in `2pc`